packages/@okta/vuepress-site/docs/guides/authenticators-okta-email/main/java/callbackuri.md

```diff
@@ -0,0 +1 @@
+`http://localhost:8080/magiclink/callback`
```

Comment:
Is this the link you use in your custom sample app or Arvind's sample app?

Reply:
FWIW this _is_ a valid link in Arvind's sample app